provider/azure/terraform/example/helpers.rb

```diff
@@ -11,13 +11,37 @@ def get_example_properties_to_check(example_name, object)
             params = param_props.map{|p| p.name.underscore}.to_set
 
             example = get_example_by_names(example_name)
-            example.properties
+            example_props = example.properties
               .reject do |pn, pv|
                 params.include?(pn) || pn == 'location'
               end
               .transform_values do |v|
                 v.is_a?(String) && !v.match(/\$\{.+\}/).nil? ? :AttrSet : v
               end
+            flatten_example_properties_to_check(example_props, true)
+          end
+
+          def flatten_example_properties_to_check(properties, is_nested)
```

Comment:
Here `is_nested`, do you mean `has_nested_item`?

Reply:
Yes, and it's changed to be `has_nested_item`